ambry-api/src/main/java/com.github.ambry/notification/CreatedBlobType.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.notification;
+
+/**
+ * Denotes the type of blob created. i.e. whether this is a simple blob, a metadata (composite) blob, or a data chunk
+ */
+public enum CreatedBlobType {
```

Comment:
Minor: `BlobType` should be fine, and should not be confused with `ContentType`.